Keas.Mvc/Startup.cs

```diff
@@ -24,6 +26,9 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            // setup entity framework
+            services.AddDbContextPool<ApplicationDbContext>(o => o.UseSqlite("Data Source=keas.db"));

```

Comment:
@srkirkland Path it so it doesn't end up in the bin directory? And add to gitignore...

Reply:
@jSylvestre it is in the gitignore now, and at least in my build it ends up in the web root and not the bin directory, which is a decent place i think.